ClearKeep/App/EnviromentKey/JoinServerEnviromentKey.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  JoinServerEnviromentKey.swift
+//  ClearKeep
+//
+//  Created by HOANDHTB on 18/08/2022.
+//
+
+import SwiftUI
+
+typealias JoinServerClosure = ((String) -> Void)?
+
+class JoinServerEnviromentKey: EnvironmentKey {
+	static let defaultValue : JoinServerClosure = { _ in }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/App/EnviromentKey/JoinServerEnviromentKey.swift#L13"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>JoinServerEnviromentKey.swift:13</code></td>
    </tr>
  </tbody>
</table>